File: C++/1019/1019.cpp
#include <cstdio>
#include <iostream>
using namespace std;

int main() {
	int time, hours, minutes, seconds;
	cin >> time;

	if (time >= 60) {
			minutes = time / 60;
			seconds = time % 60; 
	}
	else {
			minutes = 0;
			seconds = time;
	}

	if (minutes >= 60) {
			hours = minutes / 60;		
			minutes -= hours*60;
	}
	else {
			hours = 0;
	}


	printf("%i:%i:%i\n", hours, minutes, seconds);
}

File: Pascal/1019/1019.pas
uses math;
{Uses math unit so that, we can use modulus operator on double or floating data type}
var
	n : double;
	seconds, minutes, hours : double;
	hours_in_seconds, minutes_in_seconds : double;
begin
	readln(n);

	hours_in_seconds := 3600;
	minutes_in_seconds := 60;

	hours := n / hours_in_seconds;
	minutes := n mod hours_in_seconds / minutes_in_seconds;
	seconds := n mod hours_in_seconds mod minutes_in_seconds;

	writeln(trunc(hours), ':', trunc(minutes), ':', trunc(seconds));
end.
